cloudinit/config/cc_users_groups.py

```diff
@@ -78,6 +78,12 @@
     If specifying a sudo rule for a user, ensure that the syntax for the rule
     is valid, as it is not checked by cloud-init.
 
+.. note::
+    Following configurations from the above list are applicable on already
+    existing user(s).
```

Comment:
I think we want to explain a little more what's going on here; what do you think about this wording?

```suggestion
    Most of these configuration options will not be honored if the user
    already exists. These options are the exception, they will be
    applied to already-existing users:
```